config/boards/rock-5b.conf

```diff
@@ -27,16 +27,39 @@ function post_family_tweaks__rock5b_naming_audios() {
 	return 0
 }
 
-# Mainline u-boot or Kwiboo's tree
+# Mainline u-boot tree
```

Comment:
I think it's better idea to add function without branch notation and use if-return in order to avoid code duplication